src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const copyRobot = {};
+
+  for (const key of Object.keys(robot)) {
+    const value = robot[key];
+
+    if (copyRobot[value] !== undefined) {
+      return null;
+    }
```

Comment:
This is not the most reliable check for objects. Object may have pair `[key]: undefined`.
Use different method here.